CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Drop CRD management to not ensure CRDs in operators anymore.
+- Use release.Revision in Helm chart for Helm 3 support.
```

Comment:
The changelog does not work like that. `8.3.0` is already released. You need to add a new section under `Unreleased`.

Reply:
Sorry merge problem. Should be good now. PTAL.